app/controllers/dashboard.js

```diff
@@ -26,6 +27,10 @@ export default Ember.Controller.extend({
         return this.get('myFollowing').slice(0, TO_SHOW);
     }),
 
+    visibleStats: computed('myStats', function() {
```

Comment:
I'm confused, I think `myStats` is an int? Am I misunderstanding something because that's very likely